exchanges/orderbook/orderbook_test.go

```diff
@@ -294,6 +294,38 @@ func TestGetDepth(t *testing.T) {
 	}
 }
 
+func TestBaseGetDepth(t *testing.T) {
+	c, err := currency.NewPairFromStrings("BTC", "USD")
+	if err != nil {
+		t.Error(err)
+	}
+	base := &Base{
+		Pair:     c,
+		Asks:     []Item{{Price: 100, Amount: 10}},
+		Bids:     []Item{{Price: 200, Amount: 10}},
+		Exchange: "Exchange",
+		Asset:    asset.Spot,
+	}
+
+	if err = base.Process(); err != nil {
+		t.Error(err)
+	}
+
+	if result, err := GetDepth("Exchange", c, asset.Spot); err != nil {
+		t.Errorf("failed to get orderbook. Error %s", err)
+	} else if !result.pair.Equal(c) {
+		t.Errorf("Mismatched pairs: %v %v", result.pair, c)
+	}
+
+	if c, err := currency.NewPairFromStrings("BTC", "DOGE"); err != nil {
+		t.Error(err)
+	} else {
+		if _, err = GetDepth("Exchange", c, asset.Futures); !errors.Is(err, errCannotFindOrderbook) {
```

Comment:
Shouldn't this be calling `base.GetDepth()`?

Reply:
But ... then it might fail! 😉

Reply:
Fixed. Thanks! 🙏